torch/onnx/__init__.py

```diff
@@ -6,6 +6,10 @@
 
 ONNX_ARCHIVE_MODEL_PROTO_NAME = "__MODEL_PROTO"
 
+ir_version = 4
```

Comment:
If these variables are only used in the tests, we should add a TODO here, so later, in the exporter, we will also rely on this variables.